src/APIInterfaces/BackendAPI/RequestTypes/CourseRequestTypes.ts

```diff
@@ -166,4 +166,10 @@ export interface PostConfirmAttachmentUploadOptions {
     },
     studentGradeId?: number;
     studentGradeInstanceId?: number;
-}
\ No newline at end of file
+}
+
+export interface ListAttachmentOptions {
+    studentGradeId?: number;
+    studentGradeInstanceId?: number;
+    studentWorkbookId?: number;
+}
```

Comment:
NIT: Missing newline

Reply:
There is a newline, there's no 🔴 ➖ 🚫 sign like on the left side